### PR TITLE
[ADD][16.0] analytic_distribution_widget_remove_save: init

### PR DESCRIPTION
**Pull request by AnizR**
[New addon]

This module removes the save button on the widget of analytic distribution introduced by the module 'analytic'.

This button can be confusing for the end user:
The 'Save' button serves as a shortcut for quickly creating analytic
distribution templates, but it may confuse untrained users, potentially
leading to the unintended creation of templates.

widget_analytic_screenshot

### CODE REVIEW

**Review by acsonefho — APPROVED** [review]
LGTM

File: analytic_distribution_widget_remove_save/static/src/xml/analytic_distribution.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<t
+    id="analytic_distribution_inherit"
+    t-inherit="analytic.AnalyticDistributionPopup"
+    t-inherit-mode="extension"
+    owl="1"
+>
+       <xpath expr="//div[@class='popupButtons']//span" position="replace">
+           <!-- Removing button -->
+       </xpath>
```
Comment (FrancoMaxime) [inline analytic_distribution_widget_remove_save/static/src/xml/analytic_distribution.xml]:
```suggestion
       <!-- Removing button -->
       <xpath expr="//div[@class='popupButtons']//span" position="replace"/>
```

have you tried this?

**Review by FrancoMaxime — APPROVED** [review]
LGTM - minor comment

File: analytic_distribution_widget_remove_save/static/src/xml/analytic_distribution.xml (hunk 1 above)
Comment (AnizR) [reply analytic_distribution_widget_remove_save/static/src/xml/analytic_distribution.xml]: It also work, I have changed it. Thanks!

**Review by rinaldifirdaus — APPROVED** [review]
Functional test: work as expected!
image

**Review by yostashiro — APPROVED** [review]
**suggestion (non-blocking):**  It would be nice to have a CONTEXT.md to explain the background.

"The 'Save' button serves as a shortcut for quickly creating analytic distribution templates, but it may confuse untrained users, potentially leading to the unintended creation of templates."

**Comment by AnizR** [comment]
> The 'Save' button serves as a shortcut for quickly creating analytic distribution templates, but it may confuse untrained users, potentially leading to the unintended creation of templates.

Great suggestion! It's now done.

File: analytic_distribution_widget_remove_save/readme/CONTEXT.md [hunk 2]
```diff
@@ -0,0 +1 @@
+The 'Save' button serves as a shortcut for quickly creating analytic distribution templates, but it may confuse untrained users, potentially leading to the unintended creation of templates.
```
Comment (rinaldifirdaus) [inline analytic_distribution_widget_remove_save/readme/CONTEXT.md:1]: It seems have no blank line at the end of this file. could you please add it?

**Review by adrienpeiffer — APPROVED** [review]
:+1:

**Pull request merged**